meresco/dans/longconverter.py

```diff
@@ -709,15 +709,25 @@ def _getAccessRights(self, lxmlNode, e_longRoot):
                     if blnHasValidRight: break
         elif self._metadataformat.isDatacite():
             rights = lxmlNode.xpath('//datacite:resource/datacite:rightsList/datacite:rights/text()', namespaces=namespacesmap)
+            blnHasValidRight = False
             if len(rights) > 0:
                 for right in rights: # Look for the first valid edustandaard occurence
-                    blnHasValidRight = False
                     for ar in NormaliseOaiRecord.ACCESS_LEVELS:
                         if ar.lower() in right.strip().lower():
                             accessRight = ar
                             blnHasValidRight = True
                             break
                     if blnHasValidRight: break
+            if not blnHasValidRight:
+                rights = lxmlNode.xpath('//datacite:resource/datacite:rightsList/datacite:rights/@rightsURI', namespaces=namespacesmap)
```

Comment:
Dit lijkt mij het herhalen van code. Daar wordt het niet duidelijker van? Waarom gebruik je niet de (bestaande) functie: `_findFirstXpath()` ?? Dan wordt het ook makkelijker om ook de derde optie (“/@rightsIdentifier”) die in het [ticket](https://drivenbydata.atlassian.net/browse/NARCIS-1401?focusedCommentId=44345) is genoemd toe te voegen?